en-US/firefox/accounts-2018.lang

```diff
@@ -87,11 +100,11 @@ Scan the QR Code to download the Pocket App on your mobile device.
 
 
 ## TAG: accounts_2018_qrcodes
+# String is obsolete. Do not remove.
 ;Scan the QR Code to download the Lockbox App on your Apple device.
 Scan the QR Code to download the Lockbox App on your Apple device.
 
 
-# String is obsolete. Do not remove.
```

Comment:
This is no longer obsolete.

Reply:
This is only a comment, so if the string is good to use again, remove the comment. That should do.